roles/redis/templates/redis.conf.j2

```diff
@@ -790,6 +790,10 @@ acllog-max-len 128
 # AUTH <password> as usually, or more explicitly with AUTH default <password>
 # if they follow the new protocol: both will work.
 
+{% if redis_user and redis_secret %}
+user {{ redis_user }} on +@all ~* >{{ redis_secret }}
```

Comment:
extract `+@all` into `defaults/` to allow constraining the permissions

Reply:
Also: `user {{ redis_user }}` will end up as `user redis` with the name, but the container only knows about the uid.

Reply:
this is not related to the unix user - this is the user used for authentication on the redis protocol level

Reply:
https://github.com/famedly/ansible-collection-base/blob/main/roles/redis/tasks/main.yml#L60
https://github.com/famedly/ansible-collection-base/blob/main/roles/redis/tasks/main.yml#L41
all files/directories belong to this user and the container runs with this UID, so it seems to be both. is it intended that they are not decouplable right now?

Reply:
also I'm not sure if I actually want to extract this to the defaults, redis is still deployed per-service, and this was just a quick fix because netbox needs multiple stores but by default redis only sets the acl on the 0th store

Reply:
Crucially, it seems, that currently authentication with only a password (and no user) is possible. Given this change, this will be a breaking change..?

Reply:
it's not a beaking change, we are still setting the legacy password option, so login with no user won't be affected. I had the urge to rewrite the whole role when I saw how afwul it currently is, but with the current state of redis I think merging this and focusing on replacing redis with keydb in the mid-term might be a better use of resources?

Reply:
I've checked both the syntax of the change in [redis.conf.j2](https://github.com/famedly/ansible-collection-base/blob/bb0b1fb49479816241b2b7fca74f4842d7d65cd9/roles/redis/templates/redis.conf.j2#L793-L795) and also verified that [requirepass](https://github.com/famedly/ansible-collection-base/blob/bb0b1fb49479816241b2b7fca74f4842d7d65cd9/roles/redis/templates/redis.conf.j2#L798) is being ignored when an ACL is specified:
- https://stackoverflow.com/questions/66738931/how-to-set-password-for-redis-server
- https://redis.io/docs/latest/operate/oss_and_stack/management/security/acl/#when-acls-are-useful